workspaces/optic-domain/src/streams/learning-results/undocumented-endpoint-bodies.ts

```diff
@@ -15,6 +16,11 @@ export interface LearnedBody {
   rootShapeId: string;
 }
 
+export interface LearnedQueryParameters {
+  commands: any[];
```

Comment:
There's a CQRSCommands type in spectacle i believe (although I'm not sure if this will cause a circular dependency with this package)

Reply:
There is, which should probably live in `optic-domain` so it can be used everywhere else. Lets document for a follow up PR for clean up.